CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# How to contribute
+
+oria-operator is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into oria-operator.
+
+# Email and Chat
+
+- Email: [operator-framework][operator_framework]  
+
+## Getting started
+
+- Fork the repository on GitHub
+- Install Go >= `1.18`. See the [Go installation instructions][go-install] for more information.
+- Install Docker. See the [Docker installation instructions][docker-install] for more information.
+- Install KinD (Kubernetes-in-Docker). See the [KinD installation instructions][kind-install] for more information.
+
+## Reporting bugs and creating issues
+
+Reporting bugs is one of the best ways to contribute. However, a good bug report has some very specific qualities. Please review the following information on reporting bugs and creating issues:
+
+If any part of the oria-operator project has bugs or documentation mistakes, please let us know by opening an issue. We treat bugs and mistakes very seriously and believe no issue is too small. Before creating a bug report, please check that an issue reporting the same problem does not already exist.
```

Comment:
If any part of the `oria-operator` project has bugs or documentation mistakes, please let us know by opening an issue. We treat bugs and mistakes very seriously and believe no issue is too small. Before creating a bug report, please check that an issue reporting the same problem does not already exist.

Reply:
I think this is resolved with commit https://github.com/operator-framework/oria-operator/commit/c387d7632b2f440788ea0a5cb39293fb9ddbbea2